rpm/kubelet.spec

```diff
@@ -54,7 +50,7 @@ Source7: https://github.com/kubernetes-incubator/cri-tools/releases/download/v%{
 BuildRequires: systemd
 BuildRequires: curl
 Requires: iptables >= 1.4.21
-Requires: kubernetes-cni = %{CNI_VERSION}
+Requires: kubernetes-cni >= %{CNI_VERSION}
```

Comment:
b/c of line 43 above you can't use the macro here it needs to be explicit as 0.6.0 

Poke me on slack if you want to screenshare on this one.